docs/release_notes/release_notes.md

```diff
@@ -12,6 +12,28 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
 {% endif %}
 
+## Notable Features
+
+### Puppet 4 Support
+
+Puppet 4 is now supported by Katello.
+
+### Content Host Subscriptions Bulk Actions
+
+Users can now attach Subscriptions to multiple Content Hosts using the UI. 
+
+### Allow adding file type repositories to content views
+
+### Searchable ostree repos
+
+### Create/update Composite Content View by Content View names in hammer
```

Comment:
@akofink care to add some details about this feature? I know its a small one, but maybe an example command?